src/components/Timepicker/Timepicker.vue

```diff
@@ -1,18 +1,20 @@
 <template>
   <div class="time-picker">
     <i class="time-picker__icon pi pi-time-line" />
-    <fieldset class="time-picker__fieldset" :disabled="disabled">
-      <NumberInput v-model="hours" min="1" max="12" />
+    <fieldset class="time-picker__fieldset">
+      <NumberInput v-model="hours" min="1" max="12" :disabled="disabled" />
       <span class="time-picker__separator">:</span>
-      <NumberInput v-model="minutes" min="0" max="59" />
+      <NumberInput v-model="minutes" min="0" max="59" :disabled="disabled" />
```

Comment:
What was this needed for? The inputs should respect that they're in a disabled fieldset. If they don't we should fix that rather than passing in the disabled prop individually.

Reply:
I think the issue is within `NumberInput` and how it uses the `number-input--disabled` class. We should be able to wrap say 2 NumberInput components inside a `fieldset` and if the fieldset is disabled both NumberInputs should also be disabled. I'm going to look at this branch and see if I can resolve that